src/main/java/at/hannibal2/skyhanni/features/misc/customscoreboard/Events.kt

```diff
@@ -0,0 +1,531 @@
+package at.hannibal2.skyhanni.features.misc.customscoreboard
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.data.ScoreboardData
+import at.hannibal2.skyhanni.utils.LorenzUtils.inDungeons
+import at.hannibal2.skyhanni.utils.LorenzUtils.isInIsland
+import at.hannibal2.skyhanni.utils.LorenzUtils.nextAfter
+import at.hannibal2.skyhanni.utils.TabListData
+import java.util.function.Supplier
+
+private val config get() = SkyHanniMod.feature.gui.customScoreboard
+
+enum class Events(private val displayLine: Supplier<List<String>>, private val showWhen: () -> Boolean) {
```

Comment:
What events? without reading more into the code i have no idea. more exact names